src/components/Display.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+class Display extends React.Component {
+	constructor () {
+    super();
+    this.state = {
+      result : 0,
+    }
```

Comment:
* [ ] it accepts the result of the calculation as an incoming prop
 * [ ] its result prop should be a String with default value = 0
 Hi, as stated by our requirements,:
- result is coming a prop, so you don't need it in  state.
- and when its not there,  use defaultProps to set a valule of `'0'`